Lib/trufont/windows/fontWindow.py

```diff
@@ -678,13 +684,40 @@ def copy(self):
             packGlyphs = (
                 glyphs[index] for index in sorted(
                     self.glyphCellView.selection()))
+
+        fontHeight = font.info.ascender - font.info.descender
+        glyphSet = {n: font[n] for n in font.glyphOrder}
+        svgGlyphs = []
         pickled = []
+        x = 0
         for glyph in packGlyphs:
             pickled.append(glyph.serialize(
                 blacklist=("name", "unicodes")
             ))
+
+            if SVGPathPen is not None:
+                pen = SVGPathPen(glyphSet)
```

Comment:
Can we not recreate the pen? There should be a clear method or sth.

Reply:
Does it matter, most of time we are copying one glyph, and even if one is copying multiple glyphs it is unlikely to be an issue.